scripts/deploy-and-test-on-instance.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -eu
+
+JOB_UPDATE_STAGING="8d452c26-5a39-4162-b3b8-4da5efdf0b76"
+
+echo "↳ ℹ️  Deploying $COZY_APP_SLUG ($1) on $INSTANCE_TESTCAFE:"
+
+EXECUTION_ID=$(curl -s -X POST \
+     -H "X-Rundeck-Auth-Token: $RUNDECK_TOKEN" \
+     -H "Content-Type: application/json" \
+     -d "{\"argString\":\"-instance $INSTANCE_TESTCAFE -slug $COZY_APP_SLUG -source $1 \"}"  \
+     https://rundeck.cozycloud.cc/api/27/job/$JOB_UPDATE_STAGING/run | grep -o -P "(?<=execution id=').*(?=' href)")
+echo $EXECUTION_ID
+
+#init JOB_STATUS
+JOB_STATUS='running'
+
+# wait for exec to end
+while [  $JOB_STATUS == 'running' ]; do
+  sleep 1
```

Comment:
1 second is pretty agressive. 10 ? 20 ? 30 ? 60 ?

Reply:
well for what I've seen for now, it take about 4 to 6 seconds to run. So I can change to 5, but I think 10 or more will be too long!

Reply:
I think the rundeck UI can tell us what the average runtime for this job is, and we could pick a value that's about the same :)

Reply:
Agree that 10 would be too long, 5 sounds good if that's the average job runtime!

Reply:
Yep, I just checked, It's more between 5 and 8 seconds (I was a little optimistic) for the many runs I did yesterday while doing the dev

Reply:
Go for 5 or 10 ;)